setup.py

```diff
@@ -58,6 +58,7 @@
             'pytest-cov',
             'pytest-mypy',
             'pytest-runner',
+            'types-python-dateutil',
```

Comment:
Should we also mock this in docs/conf.py?

Reply:
Not needed, build https://ewatercycle--102.org.readthedocs.build/en/102/ succeeded. Also we dont import that package, its something mypy needed.